webapp/commonComponents/hooks/useProfilePicture.js

```diff
@@ -0,0 +1,31 @@
+import React, { useEffect, useState } from 'react'
+
+import User from '../../../common/user/user'
+
+import { useAsyncGetRequest } from '.'
+
+export default (userUuid, user) => {
+  const [profilePicture, setProfilePicture] = useState(null)
+
+  const uuid = userUuid || User.getUuid(user)
+
+  const {
+    data = null, dispatch,
+  } = useAsyncGetRequest(`/api/user/${uuid}/profilePicture`, {
+    responseType: 'blob',
```

Comment:
do we need responseType: 'blob'? can it be set by the server?

Reply:
According to the documentation it only affects the way the data from the server is interpreted by the browser. If you don't add it it's interpreted as text. It's already used in downloadButton.js